src/Core/Import/EntityField/Provider/CategoryFieldsProvider.php

```diff
@@ -55,23 +55,23 @@ public function getCollection()
     {
         $fields = [
             new EntityField('id', $this->trans('ID', 'Admin.Global')),
-            new EntityField('active', $this->trans('Active (0/1)')),
+            new EntityField('active', $this->trans('Active (0/1)', 'Admin.Advparameters.Feature')),
```

Comment:
wouldn't it be more logical to use `Admin.Catalog.Feature` ?

Reply:
We try to avoid duplicate the strings...